lib/policies/oauth2/oauth2-routes.js

```diff
@@ -5,9 +5,19 @@ const express = require('express');
 const passport = require('passport');
 const site = require('./site');
 const oauth2Server = require('./oauth2-server');
+const logger = require('../../logger').policy;
 
 module.exports = function (app, config) {
   app.set('view engine', 'ejs');
+  if (config.systemConfig.session.storeProvider) {
+    try {
+      const ProviderStore = require(config.systemConfig.session.storeProvider)(session);
+      config.systemConfig.session.store = new ProviderStore(config.systemConfig.session.storeOptions);
```

Comment:
If possible, I would then delete `storeProvider` and `storeOptions` once the custom provider has been created, do we do not pass this stuff down in the `express-session` middleware.